dm/master/server.go

```diff
@@ -208,32 +215,29 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 	var wg sync.WaitGroup
 	for _, stCfg := range stCfgs {
 		wg.Add(1)
-		go func(stCfg *config.SubTaskConfig) {
+		go Emit(ctx, 0, func(args ...interface{}) {
 			defer wg.Done()
-			worker, ok1 := s.cfg.DeployMap[stCfg.SourceID]
-			cli, ok2 := s.workerClients[worker]
-			if !ok1 || !ok2 {
-				workerRespCh <- &pb.CommonWorkerResponse{
-					Result: false,
-					Msg:    fmt.Sprintf("%s relevant worker not found", stCfg.SourceID),
-				}
+			cli, worker, stCfgToml, taskName, ok := s.taskConfigArgsExtractor(workerRespCh, args...)
```

Comment:
is `cli, worker, stCfgToml, taskName, err := s.taskConfigArgsExtractor(args...)` better?